.github/workflows/release.yaml

```diff
@@ -0,0 +1,219 @@
+name: PR Build
+on:
+  push:
+    tags:
+      - 'v[0-9].[0-9]+.[0-9]+'
```

Comment:
Do we want to add line anchors here? Kind of feels like better safe than sorry ... Not sure if they're implicit or not

Reply:
AFAIK, I don't think we need to worry about that? git tag names can't have ASCII control characters (e.g. newlines)

Reply:
Oh, you are worried about something like `badv1.2.3reallybad`...

Reply:
I'll add the anchors.